lib/pipedrive/resources/subscription.rb

```diff
@@ -12,6 +12,13 @@ def self.create_recurring(params)
       )
       Subscription.new(response.dig(:data))
     end
+    
+    # GET /subscriptions/find/:id
+    # Returns details of a recurring subscription by the deal ID
+    def self.find(id)
```

Comment:
I agree it's a confusing! I was keeping it consistent with the API naming, however, when we create a subscription we call that `create_recurring` rather than just `recurring` which I think is more developer friendly.